src/test/java/org/folio/rest/api/ReferenceTablesTest.java

```diff
@@ -703,6 +706,28 @@ public void statisticalCodesAndCodeTypesBasicCrud()
 
   }
 
+  @Test
+  public void instanceRelationshipTypesLoaded() throws Exception {
+    Response searchResponse = getReferenceRecords(instanceRelationshipTypesUrl(""));
+    validateNumberOfReferenceRecords("instance-relationship types",
+      searchResponse, 4, 200);
```

Comment:
If we have a test which checks for the expected count of relationship types, why does this check for a range?

Is there a reason why we aren't sure how many should be loaded?

Reply:
@marcjohnson-kint, It was a question for me as well. We use such checks for all the reference data tests. 

My guess, that maven test runner do not run tests in the same order every time and because of this there might be test cases that creates additional elements, so we just need some delta to not depend on the order.

Reply:
Ok, if these are the established patterns for this area, then any feedback I might have is likely not relevant for this pull request.

(I think that records created during tests should be deleted prior to the next test, in order to improve the reliability and specificity of tests)

Reply:
It is best practice that each test should be self contained and execution order should not matter. If order needs to matter, of course, there are ways to do it. But testing ranges instead of exact results yields a fragile test, IMHO. Tests that create things should also clean up those things. Are we not doing there here?

Reply:
@mreno-EBSCO I agree and I'd prefer it that way. 

There are quite a few places where that is not done, quite often with regards to the built in sample or reference records.

This pull request doesn't seem to me to be the place to resolve those more broader issues.